src/main.ts

```diff
@@ -165,6 +169,45 @@ const command: Command = {
 		else {
 			return compile(config(configArgs), options);
 		}
+	},
+	eject(helper: Helper) {
+		const ejectOutput: EjectOutput = {
+			npm: {
+				devDependencies: {
```

Comment:
I don't think we can use the full set of deps from the command's package.json. We likely need to pull it in and pull out the versions for a given subset.